quill-jdbc/src/test/scala/io/getquill/source/jdbc/h2/JdbcSourceSpec.scala

```diff
@@ -0,0 +1,30 @@
+package io.getquill.source.jdbc.h2
+
+import io.getquill._
+
+class JdbcSourceSpec extends Spec {
+
+  "probes sqls" - {
+    val p = testH2DB.probe("DELETE FROM TestEntity")
```

Comment:
Could you add a test cheking if it fails for invalid sqls? I'm not sure h2 supports the "PREPARE"
statement.